packages/utils/src/utils/toUtf8String.ts

```diff
@@ -47,13 +46,13 @@ function onError(
   reason: Utf8ErrorReason,
   offset: number,
   bytes: Uint8Array,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why are you removing these _eslint disable_ modifiers? They don't seem related to the PR

Reply:
No idea how this got removed :man_shrugging:
Fixed in https://github.com/FuelLabs/fuels-ts/pull/3225/commits/57781b52bfac3dd8a8af9354f0daa18d11536ad1